### PR TITLE
aws: provisioning for client VMs and README tweaks

### PR DESCRIPTION
**Pull request by ajfabbri**
- aws: readme tweaks.
- asw: readme tweaks
- aws: add AMI id for ubuntu hirsute build.
- aws: add support for provisioning blank client VMs

### CODE REVIEW

**Review by dotnwat — APPROVED** [review]
looks good to me. let's let @rkruze or @ivotron have a peek since I think they are the primary authors?

File: aws/vars.tf [hunk 1]
```diff
@@ -19,6 +19,11 @@ variable "instance_type" {
   default     = "i3.2xlarge"
 }
 
+variable "client_instance_type" {
+  description = "Default client instance type to create"
+  default     = "m5n.8xlarge"
```
Comment (rkruze) [inline aws/vars.tf:24]: This is a very beefy default client node, I would recommend something like an m5n.2xlarge.

File: aws/vars.tf [hunk 2]
```diff
@@ -30,6 +35,17 @@ variable "enable_monitoring" {
   default     = true
 }
 
+variable "clients" {
+  description = "Number of kafka client hosts to set up, if any."
+  type        = number
+  default     = 0
+}
+
+variable "client_distro" {
```
Comment (rkruze) [inline aws/vars.tf:44]: Is there a reason to have a different distro for the client vs the rest of the cluster?

**Review by rkruze — CHANGES_REQUESTED** [review]
It would also be good if rpk was installed on the client machine so that it could be useful for creating/deleting topics.

File: aws/vars.tf (hunk 1 above)
Comment (ajfabbri) [reply aws/vars.tf:24]: Ok, thanks. I picked the smallest one that had guaranteed 25 Gbps network throughput, but that is probably overkill for most basic testing. I'll add another commit to change the default here.

File: aws/vars.tf (hunk 2 above)
Comment (ajfabbri) [reply aws/vars.tf:44]: In the case that some tools/tests are easier to build on a different (newer) distro. Default is the same distro though.

**Comment by ajfabbri** [comment]
> It would also be good if rpk was installed on the client machine so that it could be useful for creating/deleting topics.

I don't add any setup of the client machines here. As mentioned in the README and commit message, these are just blank VMs for now. We probably want to add some basic ansible roles for clients here. For now I'm using other ansible scripts once I finish my VM provisioning.

**Comment by dotnwat** [comment]
> It would also be good if rpk was installed on the client machine so that it could be useful for creating/deleting topics.

@ajfabbri i think that in the ansible scripts in this project we can install the redpanda package on the client nodes

the redpanda-ansible client-testing role in the other repository shouldn't install rpk, rather we can create a resuable redpanda role at some point.

File: aws/vars.tf (hunk 1 above)
Comment (dotnwat) [reply aws/vars.tf]: would it make sense to use something like c5.4xlarge or something? the m5n nodes seem to have a lot of local storage. it's not clear we need that in the default case

**Comment by dotnwat** [comment]
i tried this an it works. what is left before this can be merged?

File: aws/vars.tf (hunk 1 above)
Comment (ajfabbri) [reply aws/vars.tf]: I'd say `c5n.xlarge` when 4 vCPUs is enough, and `m5n.2xlarge` when you want 8 cores + more memory:
image

**Comment by ajfabbri** [comment]
I have another patch and a wiki writeup waiting for this. I'm assuming y'all don't want me to take the time *now* to factor out a separate RPK installer / role and use it for the blank client machines. As is rpk is available on the RP nodes. These ansible scripts install all of RP and get it that way.

**Review by rkruze — APPROVED** [review]

**Pull request merged**